pkg/internal/checkup/executor/executor.go

```diff
@@ -257,34 +261,70 @@ func extractSectionStatistics(input, sectionStart, sectionEnd string) (string, e
 	return strings.Join(lines[startLineIdx+1:endLineIdx], "\n"), nil
 }
 
-func parseTestpmdStats(input string) (map[string]int64, error) {
-	params := make(map[string]int64)
-	scanner := bufio.NewScanner(strings.NewReader(input))
-	for scanner.Scan() {
-		line := scanner.Text()
-		if line == "" {
-			continue
+func parseTestpmdStats(input string) ([]testPmdPortStats, error) {
+	statistics := make([]testPmdPortStats, testPmdPortStatsSize)
```

Comment:
If the size is not going to change, an array may be a better fit.

Reply:
DONE